### PR TITLE
Updated landing page

### PR DESCRIPTION
**Pull request by thebe14**
Made changes to the landing page according to this document https://docs.google.com/document/d/1Xq0eLBJU3VR677p8wjl2HMLY2yNDT8J1IgsJQhqhSH8/edit

### CODE REVIEW

**Review by enolfc — APPROVED** [review]
👍

File: content/en/_index.md [hunk 1]
```diff
@@ -5,79 +5,49 @@ description: "Documentation related to EGI activities"
...
-provider to offer your innovative services. {{% /blocks/feature %}}
-
-{{% blocks/feature icon="fa-cogs" title="Internal Services" url="/internal" %}}
-Hosts the documentation of the EGI services that power the
-cloud services of EGI Federation. {{% /blocks/feature %}}
+{{< blocks/landing color="primary" height="min" title="Welcome to the EGI Documentation!">}}
+  <div>
+    <p class="lead mt-3">
+      <a href="https://www.egi.eu">EGI</a> is an international e-Infrastructure
+      that provides advanced computing and data analytics services for research
```
Comment (gwarf) [inline content/en/_index.md:12]:
```suggestion
     providing advanced computing and data analytics services for research
```

File: content/en/_index.md [hunk 2]
```diff
@@ -5,79 +5,49 @@ description: "Documentation related to EGI activities"
...
+      and innovation. The EGI Federation offers a wide range of services for compute,
+      storage, data and support.
+    </p>
+    <p class="lead mt-3">
+      This is the landing page for the user and provider documentation of the
```
Comment (gwarf) [inline content/en/_index.md:17]: I'm not really fond of the "landing page" mention, couldn't we just replace by "This is the user and provider documentation of the EGI services"?

Comment (thebe14) [reply content/en/_index.md]: Replaced it with "starting"

**Comment by gwarf** [comment]
@thebe14 thanks! Please let us know when you consider it ready for merge.

File: static/css/egi.css [hunk 3]
```diff
@@ -33,16 +42,22 @@
...
+  font-size: 14px;
   color: #3d3d3d;
 }
 
+.td-cover-block p a {
+  color: rgb(118, 182, 235);
+  font-weight: 400;
+}
+
+
```
Comment (gwarf) [inline static/css/egi.css:60]:
```suggestion
```

**Review by gwarf — APPROVED** [review]
LGTM, thanks!

**Pull request merged**